subjects/prime_checker/README.md

```diff
@@ -4,9 +4,11 @@
 
 Create a **function** `prime_checker` that takes an `u32` and check if it is a prime number.
 
-The result will be `None` if the argument is less or equal one otherwise, it will return the result with the prime number inside if the number is prime, or it will return an `enum` `PrimeErr` if the `u32` is not prime.
+The result will be `None` if the argument is less or equal one otherwise, it will return a `Result`.
```

Comment:
the `,` should be after "one" and before "otherwise".
"... ils less or equal one, otherwise it will return ..."